contentcuration/contentcuration/frontend/shared/views/StudioImmersiveModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+
+  <div
+    v-if="value"
+    class="modal-wrapper"
+    data-testid="modal-wrapper"
+    :style="{ backgroundColor: $themeTokens.surface }"
+  >
+    <KToolbar
+      textColor="white"
+      :style="{ backgroundColor: $themePalette.grey.v_900 }"
+    >
+      <template #icon>
+        <KIconButton
+          icon="close"
+          :ariaLabel="$tr('close')"
+          :tooltip="$tr('close')"
```

Comment:
Tooltip is misplaced. This is sometimes happening in Studio, we're not yet sure about the cause. Probably best to just remove `tooltip` prop. It wasn't present before anyway.